.github/workflows/rspec.yml

```diff
@@ -49,4 +49,3 @@ jobs:
         DATABASE_HOST: 127.0.0.1
         DATABASE_USERNAME: 'postgres'
         DATABASE_PASSWORD: 'test'
-        RAILS_MASTER_KEY: ${{ secrets.RAILS_MASTER_KEY }}
```

Comment:
Why did you delete this key? Make sure that you will not be using the `credentials.yml.enc` for hiding your API token